.drone.yml

```diff
@@ -180,6 +180,37 @@ pipeline:
     when:
       event: [ push, tag ]
 
+  build_docs:
+    image: webhippie/hugo:latest
+    pull: true
+    commands:
+      - cd docs
+      - make trans-copy
+      - make clean
+      - make build
+
+  docker:
+    image: plugins/docker:17.05
+    pull: true
+    secrets: [ docker_username, docker_password ]
+    repo: gitea/docs
```

Comment:
This is not correct any more?

Reply:
This would still build a separate docker image just for docs that would push to https://hub.docker.com/r/gitea/docs/ 
See the next line where I specify the docker file found in the docs directory.
This would allow the docs on the gitea site to pull from the specific docs image.

Reply:
Would these secrets prevent PRs from succeeding CI ?

Reply:
@strk good question. These secrets are already in use from the other docker container being built.

Reply:
then LGTM